lib/jsdom/living/class-list.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+var createDOMTokenList = require("./dom-token-list").createDOMTokenList;
+
+exports.createClassList = function createClassList(el) {
```

Comment:
I don't think this file (or this function) is really necessary. You can just use createDOMTokenList at the appropriate place.